configs/sst_pt_llvm_rust_go-rust-toolset.yaml

```diff
@@ -10,7 +10,12 @@ data:
     - clippy
     - rust-analyzer
     - rust-doc
+    - rust-gdb
+    - rust-lldb
     - rust-std-static
+    - rust-std-static-wasm32-unknown-unknown
+    - rust-std-static-wasm32-wasi
```

Comment:
Note that plain `wasi` will be removed in favor of `wasip1` in the near future:
https://blog.rust-lang.org/2024/04/09/updates-to-rusts-wasi-targets.html#renaming-wasm32-wasi-to-wasm32-wasip1

But for now, it's correct to include both.